### PR TITLE
Feat/refactor delete components command

### PR DESCRIPTION
**Pull request by mattcrn**
## Pull request type

Jira Link: -

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## How to test this PR

* Test the `delete-components` command
* It works the same as before

## What is the new behavior?

- Since component ids change rather frequently when deleting, pulling & pushing components I suggest to use the component name instead to create a more reliable experience for the user when using the delete command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,7 +189,25 @@ storyblok delete-components <SOURCE> --space <SPACE_ID>
...
+
+```sh
+$ storyblok push-components ./components.json --space 67819
+```
+
+Using a **path** to a multiple files
+
+```sh
+$ storyblok push-components ./page.json,../grid.json,./feature.json --space 67819
+```
```
Comment (ademarCardoso) [inline README.md:210]: The PR is from delete components and this section is from push-components, it's incorrect, right ?

File: package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "storyblok",
-  "version": "3.15.0",
+  "name": "towa-storyblok",
+  "version": "3.16.1",
   "description": "A simple CLI to start Storyblok from your command line.",
   "repository": {
     "type": "git",
-    "url": "https://github.com/storyblok/storyblok-cli.git"
+    "url": "https://github.com/towa-digital/storyblok-cli.git"
```
Comment (ademarCardoso) [inline package.json:7]: Revert this changes please

**Comment by mattcrn** [comment]
I have moved this changes to this pr: https://github.com/storyblok/storyblok-cli/pull/46
